### PR TITLE
Upgraded to Angular 8

### PR DESCRIPTION
**Pull request by basvermeulen**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -10,34 +10,34 @@
...
-    "@types/node": "^7.0.2",
+    "@angular/animations": "^8.0.0",
+    "@angular/cdk": "^8.0.0",
+    "@angular/common": "^8.0.0",
+    "@angular/compiler": "^8.0.0",
+    "@angular/compiler-cli": "^8.0.0",
+    "@angular/core": "^8.0.0",
+    "@angular/forms": "^8.0.0",
+    "@angular/material": "^8.0.0",
+    "@angular/platform-browser": "^^8.0.0",
```
Comment (mgechev) [inline package.json]: There's a double `^` here

File: lib/pipe-symbol.ts [hunk 2]
```diff
@@ -59,7 +59,7 @@ export class PipeSymbol extends Symbol {
    * @memberOf PipeSymbol
    */
   getMetadata(): Pipe | null {
-    return this.resolver.resolve(resolveForwardRef(this.symbol));
+    return this.resolver.resolve(resolveForwardRef(this.symbol as any));
```
Comment (mgechev) [inline lib/pipe-symbol.ts]: Why do we have to cast to `any` here?

Comment (basvermeulen) [reply lib/pipe-symbol.ts]: The following commit changed the signature of 'resolveForwardRef' between version 7.0.0 and 7.1.x. 

https://github.com/angular/angular/commit/b0476f308b5a33ee1e6479e2c602c7b01689420f

I'd actually like to propose to change the code to:
```
return this.resolver.resolve(resolveForwardRef<any>(this.symbol));
```

File: package.json (hunk 1 above)
Comment (basvermeulen) [reply package.json]: Sloppy of me.

**Comment by basvermeulen** [comment]
No idea what went wrong here but it seems like this PR is squashed and merged into master and the commit message of https://github.com/mgechev/ngast/commit/331fd59324ac0b06fb480979020197d5b5eb98cf is used.

Could we alter the message of commit https://github.com/mgechev/ngast/commit/ba21673a74f046422184ad2eff2090d8fcc8218a to 'upgraded to Angular 8'?

**Pull request merged**